changelogs/changelog.yaml

```diff
@@ -673,3 +673,51 @@ releases:
       name: sonic_stp
       namespace: ''
     release_date: '2024-01-03'
+  2.4.0:
+    changes:
+      bugfixes:
+      - requirements - Update requires_ansible version in meta/runtime.yml to the
+        oldest supported version (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/321).
+      release_summary: '| Release Date: 2024-0108
+
+        | This release provides an Ansible compliance change required on top of the
+        changes included in
+
+        | the 2.3.0 release of the enterprise_sonic Ansible network resource module
+        collection.
+
+        | It addresses an issue raised by the Ansible core team with the content of
+        the 2.3.0 release,
+
+        | and provides accompanying documentation changes in the README file. Additional
+        details are
+
+        | described below.
+
+        | 1) Update the "requires_ansible" version in the meta/runtime.yml file for
+        this collection
+
+        | to the oldest supported version of ansible-core. (This was recently changed
+        by Redhat/Ansible
+
+        | to version 2.14.0.)
```

Comment:
```suggestion
        | to version 2.14.0)
```

Reply:
I have enclosed the version number in quotes to separate it from the period at the end of the sentence. I have done this for the Ansible version and Python version mentioned in this release note (for both the changelogs/changelog.yaml file and, via automatic update, the CHANGELOG.rst file).